cmd/system-probe/common/common_nix.go

```diff
@@ -7,7 +7,30 @@
 
 package common
 
+import (
+	ebpfkernel "github.com/DataDog/datadog-agent/pkg/security/ebpf/kernel"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
 const (
 	//nolint:revive // TODO(EBPF) Fix revive linter
 	DefaultLogFile = "/var/log/datadog/system-probe.log"
 )
+
+// DisablePESUnsupportedKernel returns true if network_process needs to be disabled due to unsupported kernel version
+func DisablePESUnsupportedKernel(isEnabled bool) bool {
```

Comment:
Bunch of nits (as always with nits, these don't need to be addressed if you disagree or need to merge): 
* the name and signature of the function is `DisableX(bool)` which implies that this function is actually disabling based on the input boolean.  A better name would be `CanEnablePES`
* the isEnabled parameter might not be necessary, a boolean parameter can be a code smell.  This article does a good job explaining why https://alexkondov.com/should-you-pass-boolean-to-functions/ 
* The name could be phrased as `enabled` not `disabled` so readers don't have to resolve double negative ("is pes not disabled/is pes disable" vs "is ped enabled/is pes not enabled")

Reply:
thanks for the feedback and article... I named it different in tracer.go and changed it a bit lmk what you think

Reply:
think I addressed this, unresolve if I didn't